landingai/pipeline/image_source.py

```diff
@@ -337,3 +338,35 @@ def __next__(self) -> "FrameSet":
 
     class Config:
         arbitrary_types_allowed = True
+
+
+class Webcam(NetworkedCamera):
```

Comment:
Thanks for the contribution!. Our goal is to support CLI, Jupyter and Collab. We had a half backed function which we use on a couple of notebooks. Can you please try to merge it with https://github.com/landing-ai/landingai-python/blob/7f80849f4c19ed4aa3ae34947e182d08163eda71/landingai/image_source_ops.py#L74

You will find there the code needed for collab. Thanks!!

Reply:
The main goal here is to show to the general public an easy way to use local webcam as one would use an RTSP camera though `NetworkedCamera`.

In order to support also Collab, we would need to refactor a good amount of things here (to support FPS, motion detection and other features provided by the `NetworkedCamera`). I think we can postpone this to a future version.

Added a comment highlighting that `Webcam` is not supported on Collab and remote Jupyter notebook

Reply:
True, making a browser camera equivalent to the networkcamera will require work. Still, this may be exactly what Dillon needs for the Oct 3 demo.... I just asked him and CC you

Have you checked if it is feasible to get a video feed from a webcam in jupyter and collab??  (I never checked)

Reply:
Based on dillon's comment I think we are ok with your changes. Please just add a note on the doc mentioning that for collab we have the other function. Thanks!